### PR TITLE
[Navi3x Bug Fix] fix typo to accept MNKPadding flag correctly.

### PR DESCRIPTION
**Pull request by aska-0096**
As stated in the title. This PR fix a blocking issue on AIT side.

cc. @Yanxing-Shi

### CODE REVIEW

File: include/ck/tensor_operation/gpu/device/impl/device_gemm_multiple_d_wmma_cshuffle.hpp [hunk 1]
```diff
@@ -105,7 +105,7 @@ struct DeviceGemmMultipleD_Wmma_CShuffle : public DeviceGemmMultipleD<ALayout,
 #endif
         }();
 
-        if constexpr(GemmSpec == GemmSpecialization::MNPadding)
+        if constexpr(GemmSpec == GemmSpecialization::MNKPadding)
```
Comment (asroy) [inline include/ck/tensor_operation/gpu/device/impl/device_gemm_multiple_d_wmma_cshuffle.hpp:108]: This doesn't look right.

I don't see K is being packed

**Review by asroy — CHANGES_REQUESTED** [review]

File: include/ck/tensor_operation/gpu/device/impl/device_gemm_multiple_d_wmma_cshuffle.hpp (hunk 1 above)
Comment (aska-0096) [reply include/ck/tensor_operation/gpu/device/impl/device_gemm_multiple_d_wmma_cshuffle.hpp]: A fix commit landing, it utilize the `MatrixPadder` struct to help padding work in wmma device op.

File: example/01_gemm/gemm_wmma_fp16.cpp [hunk 2]
```diff
@@ -19,7 +19,7 @@ using AElementOp = PassThrough;
 using BElementOp = PassThrough;
 using CElementOp = PassThrough;
 
-static constexpr auto GemmDefault = ck::tensor_operation::device::GemmSpecialization::Default;
+static constexpr auto GemmDefault = ck::tensor_operation::device::GemmSpecialization::MNKPadding;
```
Comment (asroy) [inline example/01_gemm/gemm_wmma_fp16.cpp:22]: should be:

static constexpr auto GemmMNKPadding = ck::tensor_operation::device::GemmSpecialization::MNKPadding;

**Review by asroy — CHANGES_REQUESTED** [review]
need cosmetic change, otherwise looks good

**Review by asroy — APPROVED** [review]

**Pull request merged**